packages/components/src/item-group/styles.ts

```diff
@@ -23,6 +23,16 @@ export const unstyledButton = css`
 	&:hover {
 		color: ${ COLORS.ui.theme };
 	}
+
+	&:focus {
```

Comment:
Have you considered using [`:focus-visible`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) instead, so that the focus ring only shows when interacting with the keyboard?

Reply:
That could be something to keep in mind, yes. Let's wait for Joen's feedback , in case :)

Reply:
Hey, thanks for the ping! Just for clarity, here's before:

![before 1](https://github.com/WordPress/gutenberg/assets/1204802/77c2ae48-4246-4ae6-aa49-f060a05b447d)

![before 2](https://github.com/WordPress/gutenberg/assets/1204802/8731fe40-86be-4f5a-bed3-110cdc6ea064)

Here's after:

![after 1](https://github.com/WordPress/gutenberg/assets/1204802/dec40d72-ec9d-4ca9-9172-0d8fa831a99a)

![after 2](https://github.com/WordPress/gutenberg/assets/1204802/d6a912ee-25d4-44bf-bf5e-e1f1f5f4cb2c)

In the after state, the focus style is now the correct color and the correct width (1.5px) in both places. However to Marco's point, it also now adds sticks the focus style when mouse navigating, which is especially jarring in these drilldown situations. I'd lean towards using focus visible here, it's already being applied for the back button in the site view, despite that lingering tooltip.

Reply:
Thanks both! I agree that `:focus-visible` makes more sense here, I've opened up a small follow-up PR: https://github.com/WordPress/gutenberg/pull/51787